docs/ccoctl.md

```diff
@@ -215,3 +215,124 @@ This command will delete the service id from the IBM Cloud
 ```bash
 ccoctl ibmcloud delete-service-id --credentials-requests-dir <path-to-directory-with-list-of-credentials-requests> --name <name> 
 ```
+
+## Alibaba Cloud
+
+This is a guide for using manual mode on alibaba cloud, for more info about manual mode, please refer to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+For alibaba cloud,  the CCO utility (`ccoctl`) binary will create credentials Secret manifests for the OpenShift installer. It will also create a user along with a long-lived RAM AccessKey (AK) for each OpenShift in-cluster component. Every RAM user who owns the AK would be attached RAM policies with the permission defined in each component's CredentialsRequest. To do all this, ccoctl consumes AK of a root RAM user with permissions required for creating user/policy and also attaching the policy to the user.
+
+### Prerequisite
+
+1. Extract and prepare the ccoctl binary from the release image.
+
+2. Choose an existing RAM user who has the below permissions, and get this user's accesskey id/secret for creating the RAM users and policies for each in-cluster component.
+
+   ```
+ram:CreatePolicy
+ram:GetPolicy
+ram:CreatePolicyVersion
+ram:DeletePolicy
+ram:DetachPolicyFromUser
+ram:ListPoliciesForUser
+ram:AttachPolicyToUser
+ram:CreateUser
+ram:GetUser
+ram:DeleteUser
+ram:CreateAccessKey
+ram:ListAccessKeys
+ram:DeleteAccessKey
+   ```
+
+3. Use the selected RAM user’s accesskey id/secret to configure the Alibaba Cloud SDK client's credential provider chain with [Envionment Creadentials](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#1-environment-credentials) mode or through [Credentials File](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file) mode
```

Comment:
Are these links official Alibaba Cloud documentation intended for the end user?

Reply:
Yes, this sdk is our official supported, and these docs are also for end user consumption.

Reply:
Thanks! :+1:

Reply:
@DahuK is there something more official? When we downstream these docs; we don't want to link to github; if at all possible.